### ISSUE TITLE
Colors aren't reflected until a state change when used with card-mod 3.0

### ISSUE THREAD

**Issue by rccoleman** [custom-cards/button-card#440]
**Checklist**
- [x] I updated the card to the latest version available
- [x] I cleared the cache of my browser

**Describe the bug**

I ran some more experiments and this seems to be caused by the newly-released Card Mod 3.0.  I didn't see a relationship between it and your card because this behavior happens without any card_mod styling, but there seems to be some interaction.  I'll file an issue there, but I'd like to leave this open for now in case Thomas punts it back.

---------

Colors assigned to buttons aren't reflected until the associated entity changes state when card-mod 3.0 is installed, regardless of whether any card-mod styling is applied to the custom button card.  After a refresh, whether or not the cache is cleared, the icon will be white until its state changes.  The initial color was working fine prior to updating to card-mod 3.0 and even before I cleared the browser cache after updating, but I start seeing the new behavior after clearing the browser cache with the latest HA beta, and on both desktop and my iPhone.  There seems to be some interaction between card-mod 3.0 and custom button card that's causing this.

**Version of the card**
Version: 3.5.0-dev.1 & 3.4.2

**To Reproduce**
This is the configuration I used:
```yaml
type: 'custom:button-card'
entity: input_boolean.test1
state:
  - value: 'off'
    operator: ==
    color: green
  - operator: default
    color: red
```

or even

```
type: 'custom:button-card'
entity: input_boolean.test1
color: green
```

**Screenshots**

After reload:
![image](https://user-images.githubusercontent.com/860888/109244730-0f455800-7794-11eb-8a23-aada71f159bf.png)

After I click the image to cause the underlying input_boolean to change state:
![image](https://user-images.githubusercontent.com/860888/109245331-1b7de500-7795-11eb-8447-7b31fd2761e5.png)

**Expected behavior**
I expect the color to be reflected immediately when the page is displayed or after a reload, as it has been up until now.

**Desktop (please complete the following information):**
 - Browser: Chrome (Windows)
 - Version: 88.0.4324.182 (Official Build) (64-bit)

**Smartphone (please complete the following information):**
 - Device: iPhone XS Max
 - OS: iOS 14.4
 - Browser Safari, the iOS HA app
 - Version ??

**Additional context**

**Comment by RomRider** [comment]
If it has worked until now and card-mod breaks it, why don't you ask card-mod's dev to fix it on their side?

card-mod plays with CSS after the element is created, not button-card 🤷‍♂️ if it's broken, I don't think it's not on me.

You can keep it opened of you want. I will have a look if I have time,but until then open an issue over there.

**Comment by rccoleman** [comment]
I did open an issue over there.

**Comment by thomasloven** [comment]
This has been fixed in card-mod 3.0.1.

Sorry about the inconvenience.
